lib/CMakeLists.txt

```diff
@@ -10,13 +10,13 @@ include_directories(
 set(SOURCES
     src/connector/client_metadata.cc
     src/connector/connection.cc
-    src/connector/connector.cc
     src/connector/session_association.cc
```

Comment:
Why not moving `session_association.cc` to `connector/v1`?
Does it have to do with the fact that Session Association will be removed from PCP v2 (PCP-679)?